common/traits/hfe_species_trait.txt

```diff
@@ -33,78 +34,128 @@ hfe_species_trait_war_fans = {
 	leader_age_min = 10
 	icon = gfx/interface/icons/other/hfe_war_fans.dds
 	allowed_archetypes = {
-		BIOLOGICAL
+		BIOLOGICAL LITHOID
 	}
 	ai_weight = {
 		weight = 100
 	}
 	modifier = {
 		admiral_skill_levels = 1
 		general_skill_levels = 1
-		hfe_planet_jobs_war_produces_mult = 0.3
```

Comment:
No

Reply:
That was creating Error Logs. I had to get rid of it.

Reply:
Make it commented out

Reply:
Okay, done